source/_components/insteon_local.markdown

```diff
@@ -0,0 +1,51 @@
+---
+layout: page
+title: "Insteon(local)"
+description: "Instructions how to setup the Insteon Hub locally within Home Assistant."
+date:  2016-12-18 08:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: insteon.png
+ha_category: Hub
+ha_iot_class: "Local Polling"
+---
+
+The `insteon-local` component lets you use your [Insteon Hub](http://www.insteon.com/insteon-hub/) with Home Assistant.
+
+To integrate your Insteon(local) with Home Assistant, add the following section to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml platform entry
+insteon_local:
+  host: YOUR HUB IP
+  username: YOUR HUB USERNAME
+  password: YOUR HUB PASSWORD
+# Example configuration.yaml light entry  
+light:
```

Comment:
Light and switch examples should be excluded from this file.
Optionally have a basic example and a full configuration example.